data/set/finite.lean

```diff
@@ -287,4 +289,10 @@ assume hf,
 have injective (f ∘ (coe : ℕ → ℤ)), from injective_comp hf $ assume i j, int.of_nat_inj,
 not_injective_nat_fintype this
 
+instance (α : Type u) [fintype α] : fintype (set α) :=
+fintype.of_bijective finset.to_set
+⟨λ _ _, finset.coe_eq_coe.1,
+λ x, by haveI := classical.prop_decidable;
+  exact ⟨set.finite.to_finset ⟨set_fintype _⟩, finset.coe_to_finset⟩⟩
+
```

Comment:
Just realised this could be done with `pi.fintype`, since the `decidable_eq Prop` instance that the definition requires isn't actually used. Would this be preferable? @digama0

Reply:
Yes. (I recently changed the definition to remove that redundant parameter.)

Reply:
Done, although there's a bit of confusion, because it now depends on the latest mathlib. Not sure it's going to build come to think of it.

Reply:
You can fix this by rebasing the branch onto the latest master. That is, check out your branch and then `git rebase origin/master`.

Reply:
After a lot of fiddling around with git, I think I've managed to sort it out.